Sources/Event.swift

```diff
@@ -861,6 +861,63 @@ extension Signal.Event {
 			}
 		}
 	}
+	
+	internal static func collect(every interval: DispatchTimeInterval, on scheduler: DateScheduler, skipEmpty: Bool, discardsWhenCompleted: Bool) -> Transformation<[Value], Error> {
+		return { action, lifetime in
+			let values = Atomic<CollectEveryState<Value>>(.init(skipEmpty: skipEmpty))
```

Comment:
Could we also rename `values` to `state` since we have to add the `dispose` call below?